local.tf

```diff
@@ -44,46 +44,39 @@ locals {
+    ami_id                                   = "${data.aws_ami.eks_worker.id}" # AMI ID for the eks workers. If none is provided, Terraform will search for the latest version of their EKS optimized worker AMI.
+    asg_desired_capacity                     = "1"                             # Desired worker capacity in the autoscaling group.
+    asg_max_size                             = "3"                             # Maximum worker capacity in the autoscaling group.
+    asg_min_size                             = "1"                             # Minimum worker capacity in the autoscaling group.
+    asg_force_delete                         = false                           # Enable forced deletion for the autoscaling group.
+    instance_type                            = "m4.large"                      # Size of the workers instances.
+    override_instance_type                   = "t3.large"                      # Need to specify at least one additional instance type for mixed instances policy. The instance_type holds  higher priority for on demand instances.
+    on_demand_allocation_strategy            = "prioritized"                   # Strategy to use when launching on-demand instances. Valid values: prioritized.
+    on_demand_base_capacity                  = "0"                             # Absolute minimum amount of desired capacity that must be fulfilled by on-demand instances
+    on_demand_percentage_above_base_capacity = "100"                           # Percentage split between on-demand and Spot instances above the base on-demand capacity
+    spot_allocation_strategy                 = "lowest-price"                  # The only valid value is lowest-price, which is also the default value. The Auto Scaling group selects the cheapest Spot pools and evenly allocates your Spot capacity across the number of Spot pools that you specify.
+    spot_instance_pools                      = 10                              # "Number of Spot pools per availability zone to allocate capacity. EC2 Auto Scaling selects the cheapest Spot pools and evenly allocates Spot capacity across the number of Spot pools that you specify."
+    spot_max_price                           = ""                              # Maximum price per unit hour that the user is willing to pay for the Spot instances. Default is the on-demand price
+    spot_price                               = ""                              # Cost of spot instance.
+    placement_tenancy                        = "default"                       # The tenancy of the instance. Valid values are "default" or "dedicated".
+    root_volume_size                         = "100"                           # root volume size of workers instances. Only used for cluster-autoscaler tags on ASGs. You cannot change the root volume size with a launch template.
```

Comment:
> You cannot change the root volume size with a launch template.

How can this be? You can change this for any AMI using an LC. Are you saying this option is simply impossible with LTs?